packages/ethernaut-optigov/README.md

```diff
@@ -0,0 +1,47 @@
+# ethernaut-optigov
+
+Tasks for interacting the Optimism retroactive public goods funding rounds (RetroPGF)
```

Comment:
Wouldn't the scope of the plugin be more than this? I.e. the Agora API would also allow this plugin to access general Optimism governance, like proposals, delegation, etc.

Reply:
ethernaut-optigov plugin description updated